b2b-sample/libs/business-admin-app/data-access/data-access-controller/src/lib/controller/controllerSwitchOrg/controllerDecodeSwitchOrg.ts

```diff
@@ -20,26 +20,20 @@ import { commonControllerDecode } from "@b2bsample/shared/data-access/data-acces
 import { OrgSession } from "@b2bsample/shared/data-access/data-access-common-models-util";
 import { JWT } from "next-auth/jwt";
 import { controllerCallSwitchOrg } from "./controllerCallSwitchOrg";
-import config from "../../../../../../../../config.json";
 
 function getOrgId(token: JWT): string {
 
     if(token.user) {
         if (token.user.user_organization) {
 
             return token.user.user_organization;
-        } else if (config.CommonConfig.ApplicationConfig.SampleOrganization[0]) {
-    
-            return config.CommonConfig.ApplicationConfig.SampleOrganization[0].id;
-        } else {
-    
```

Comment:
Shall we change the order of the config check rather removing them?

Reply:
Sure, based on the conclusion of this discussion https://github.com/wso2/samples-is/pull/405/files#r1133118204, will attend to this